test/test_jsonclient.rb

```diff
@@ -24,6 +24,14 @@ def test_post
     assert_equal(1, JSON.parse(res.previous.content)['a'])
   end
 
+  def test_post_with_array
+    res = @client.post(serverurl + 'json', [{'a' => 1}, {'b' => {'c' => 2}}])
+    assert_equal(2, res.content[1]['b']['c'])
+    assert_equal('application/json; charset=utf-8', res.content_type)
+    # #previous contains the original response
```

Comment:
@elifoster good point. I deferred to the style of the test above, but I'm happy to remove the dup #